spec/requests/conversations/update_spec.rb

```diff
@@ -0,0 +1,58 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe 'PUT /conversations/:id', type: :request do
+  let(:conversation) { create :conversation_with_messages, unread: true }
+  let(:params) do
+    {
+      conversation: {
+        unread: unread
+      }
+    }
+  end
+  let(:user) { conversation.initiator }
+
+  subject do
+    put "/conversations/#{conversation.id}", params: params, headers: headers
```

Comment:
you can do: `put conversation_put_method_path(id: conversation.id)...`

Reply:
I feel like this way I'm also testing the routes. So if a change is made in the routes file and involves any of these conversations endpoints, I should be able to catch any change when running this spec. What do you think? Should I change it to use the rails helper methods anyways?

Reply:
I'm not sure. There are specs for routes (I never did one) but if you are worry about those one, you can do it. I have seen those specs in Alexis projects. 

Anyway when changing the routes.rb, the rails helper method will change (probably).
i.e: If you add a new namespace, the namespace will be present in the helper method name

Reply:
Ahh you're right. The helper method name will also change. Unless the change was made using extra options in the route definition. I think it's ok then, I'll start to use these helpers in the next pull requests.